src/components/ContactList.tsx

```diff
@@ -92,8 +92,12 @@ const ContactList = () => {
                     value={searchInput} 
                     onChange={(e) => setSearchInput(e.target.value)}
                 />
-                <button type="button" onClick={handleClick}>Search</button>
-                <Link to="/contacts">See All Contacts</Link>
+                <button id="search-button" type="button" onClick={handleClick}>Search</button>
+            </div>
+            <div id="button-div">
+                <button type="button"><Link to="/create">Create New Contact</Link></button>
+                <button type="button"><Link to="/contacts">See All Contacts</Link></button>
```

Comment:
Don't recommend `<Link>` inside a button. Instead, I recommend using the `useNavigate` hook provided in React Router

```suggestion
                <button type="button" onClick={() => navigate('/create')}>Create New Contact</button>
                <button type="button" onClick={() => navigate('/contacts')}>See All Contacts</Link></button>
```

https://reactrouter.com/en/main/hooks/use-navigate

Reply:
Good call. I have experience with useNavigate so this will be no problem.